monitoring/nethealth.go

```diff
@@ -121,7 +121,7 @@ func (c *nethealthChecker) Check(ctx context.Context, reporter health.Reporter)
 func (c *nethealthChecker) check(ctx context.Context, reporter health.Reporter) error {
 	addr, err := c.getNethealthAddr()
 	if trace.IsNotFound(err) {
-		log.WithError(err).Warn("Nethealth pod was not found.")
+		log.WithError(err).Debug("Nethealth pod was not found.")
```

Comment:
I would honestly get rid of `.WithError(err)` as well - the stack trace is very verbose, so even at debug level not much benefit in seeing the whole stack trace for the not found error here.